pkg/secret/replicated.go

```diff
@@ -22,7 +23,7 @@ import (
 )
 
 const (
-	passwordCharacters = "!#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~"
+	passwordCharacters = "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
```

Comment:
This is temporary-ish, see https://github.com/thelastpickle/cassandra-reaper/pull/1144 for context.